docs/src/Layer.doc.js

```diff
@@ -42,6 +42,62 @@ card(
   />
 );
 
+card(
+  <Example
+    description="
+    Since Layer is rendered within a portal element being manually added to the DOM, it is supposed to be rendered and dismissed only once. 
+    Any re-renders triggered by the parent will effectively remove the portal element from the DOM, hence it won't be re-created and internal elements like Sheet may not reappear.
+    The example below demonstrate a technique to avoid Layer re-rendering by making sure parent re-renders don't force re-renders by passing useCallback function props and preventing re-renders with memo().
+  "
+    name="Avoid Re-rendering"
+    defaultCode={`
+function Example() {
+  // We're using memo() here to prevent re-renders similarly to
```

Comment:
Another run-on that can be broken up:
"We're using memo() here to prevent re-renders. This is similar to shouldComponentUpdate(), but for functional components."

Reply:
Suggestion accepted, thank you!